hack/util.sh

```diff
@@ -301,15 +306,15 @@ function wait_for_url_timed {
 	prefix=${2:-}
 	max_wait=${3:-10*TIME_SEC}
 	wait=0.2
-	expire=$(($(time_now) + $max_wait))
+	expire=$(($(time_now) + max_wait))
 	set +e
 	while [[ $(time_now) -lt $expire ]]; do
-	out=$(curl --max-time 2 -fs $url 2>/dev/null)
+	out=$(curl --max-time 2 -fs "$url" 2>/dev/null)
 	if [ $? -eq 0 ]; then
 		set -e
-		echo ${prefix}${out}
+		echo "${prefix}${out}"
```

Comment:
Right, this line is to be blamed for most of the spam that happens whenever we start the server using these scripts...